### DISCUSSION TITLE
Creating Feature Branches For the Mortgage and Currency Conversion Calculator

### DISCUSSION THREAD

**Discussion by Chanmonich-Phe** [Chanmonich-Phe/Tafesa22S202#3] [General]
Hello Prince,

We're going to continue our process by creating the two feature branches called Mortgage Calculator and the other one is Currency Conversion Calculator. I'll leave you to do your Currency Conversion Calculator while I'll work on the Mortgage Calculator. Please follow the project instruction. If there's any questions, feel free to ask by joining the discussion!

Cheers,
Chanmonich

**Comment by PrinceArmah01** [comment]
Oh thanks for reaching out! I don't have any question at the moment.

**Comment by PrinceArmah01** [comment]
Oh I forgot to ask about the C# coding, is there a specific naming conventions that you want me to follow for the project?

**Comment by Chanmonich-Phe** [comment]
Hello Prince,

Yes thanks for asking, sorry for not specifying on that! Though you don't have to follow any naming conventions for the C# coding. Feel free to name it however you want but please be reasonable as well while naming them.

**Comment by Chanmonich-Phe** [comment]
Hello Prince,

Just messaging to catch up with how you are doing with the branching and coding? Did you encounter any issues?

**Reply by PrinceArmah01** [reply]
Hello Monich,

Actually everything is going pretty good I just had a few issues with the GUID in C# since we were both accessing the same C# file but I fixed it now and yeah all good. Thanks for reaching out.

**Reply by Chanmonich-Phe** [reply]
That's good, Prince! Well Done!

**Comment by PrinceArmah01** [comment]
Hello Monich, 

I am done with the currency conversion calculator in the currencyConversionCalculator branch, I just merged it into the development branch and pushed to the  remote development branch on Github.

**Comment by Chanmonich-Phe** [comment]
Oh that's wonderful! I just finished merging everything although I ran into some conflicts; I was able to resolve it. So yup all good since I tested the application and everything is running fine. Good Work!!
